src/lib/forms/helpers.ts

```
@@ -531,3 +531,420 @@
+			(acc, [key, value]: [string, CustomRank]) => ({
+				...acc,
+				[`audio_${key}_fetch`]: value.fetch,
+				[`audio_${key}_use_custom_rank`]: value.use_custom_rank,
+				[`audio_${key}_rank`]: value.rank
+			}),
+			{}
+		),
+		// Extras
+		...Object.entries(data.ranking.custom_ranks.extras).reduce(
+			(acc, [key, value]: [string, CustomRank]) => ({
+				...acc,
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)